doc/mgr/orchestrator.rst

```diff
@@ -550,6 +550,25 @@ Or with hosts:
         - host2
         - host3
 
+Updating Service Specifications
+===============================
+
+The Ceph Orchestrator maintains a declarative state of each
+service in a ``ServiceSpec``. For certain operations, like updating
+the RGW HTTP port, we need to update the existing
+specification.
+
+First, get the current ``ServiceSpec``::
```

Comment:
In this instance `get` means to list? yes? If so, please say "list" instead of get.

Reply:
```suggestion
1. List the current ``ServiceSpec``::

Reply:
fixed